contracts/token/ERC721/ERC721.sol

```diff
@@ -269,34 +280,37 @@ abstract contract ERC721 is Context, ERC165, IERC721, IERC721Metadata, IERC721Er
      *
      * Emits a {Transfer} event.
      */
-    function _mint(address to, uint256 tokenId) internal virtual {
+    function _mint(address to, uint256 tokenId) internal {
         if (to == address(0)) {
             revert ERC721InvalidReceiver(address(0));
         }
-        if (_exists(tokenId)) {
+        address from = _update(to, tokenId, address(0));
```

Comment:
We should use the same variable name here, either `owner` or `previousOwner`.

Reply:
I'd be in favor of `previousOwner`

Reply:
changed to previousOwner

Reply:
what about in the extensions like ERC721Enumerable

```solidity
    function _update(address to, uint256 tokenId, address auth) internal virtual override returns (address) {
        address from = super._update(to, tokenId, auth);

        if (from == address(0)) {
            _addTokenToAllTokensEnumeration(tokenId);
        } else if (from != to) {
            _removeTokenFromOwnerEnumeration(from, tokenId);
        }
        if (to == address(0)) {
            _removeTokenFromAllTokensEnumeration(tokenId);
        } else if (from != to) {
            _addTokenToOwnerEnumeration(to, tokenId);
        }

        return from;
    }
```

Reply:
I would rather be consistent in there too.

Reply:
done